### ISSUE TITLE
Engines shutting down too slow

### ISSUE THREAD

**Issue by MCCVeen** [mSparks43/747-400#542]
For some reason whenever I shut the engines down the N1 decreases way too slowly. It used to be fine but now they take forever to slow down. See this video for reference of the CF6: https://youtu.be/w2BvSnWvCA0?t=1080

**Comment by MCCVeen** [comment]
@mSparks43 Could you have a look at this before the 2nd release? 

Another reference here of the CF6: https://youtu.be/7GZ3fSDFevE?t=337

https://user-images.githubusercontent.com/75425109/119750022-ad7eb200-be98-11eb-8e43-0ad7ce42c68b.mp4

Can't find a PW example but it's pretty much the same, I'd say revert to the old shutdown rate

**Comment by mSparks43** [comment]
are you talking about n1 or when the power from the gen cuts? I'm not seeing a big difference to those videos, and afaik they havent been touched (nor can they)

**Comment by MCCVeen** [comment]
> are you talking about n1 or when the power from the gen cuts? I'm not seeing a big difference to those videos, and afaik they havent been touched (nor can they)

they have been touched, just compare it to the default 747 and you'll see that they shut down faster there which is correct

**Comment by mSparks43** [comment]
the default engines have something like a 5 foot diameter, absolutely nothing is right about them.

**Comment by crazytimtimtim** [comment]
Speaking of default stuff, does the Sparky use the default flight model or MGouge's mod (on the org)? We might be able to get permission to copy his mod into the Sparky if not. It'll be tricky though because so much else has changed in the acf...

**Comment by crazytimtimtim** [comment]
> Speaking of default stuff, does the Sparky use the default flight model or MGouge's mod (on the org)? We might be able to get permission to copy his mod into the Sparky if not. It'll be tricky though because so much else has changed in the acf...

@mSparks43

**Comment by MCCVeen** [comment]
> are you talking about n1 or when the power from the gen cuts? I'm not seeing a big difference to those videos, and afaik they havent been touched (nor can they)

n1 btw

**Comment by MCCVeen** [comment]
> the default engines have something like a 5 foot diameter, absolutely nothing is right about them.

but still the rate at which the engines shut down is way more realistic

**Comment by MCCVeen** [comment]
> @mSparks43 Could you have a look at this before the 2nd release?
> 
> Another reference here of the CF6: https://youtu.be/7GZ3fSDFevE?t=337
> 
>  2021-05-27.03-02-42.mp4 
> Can't find a PW example but it's pretty much the same, I'd say revert to the old shutdown rate

Can this be looked at? The evidence is here.

**Comment by mSparks43** [comment]
> > @mSparks43 Could you have a look at this before the 2nd release?
> > Another reference here of the CF6: https://youtu.be/7GZ3fSDFevE?t=337
> > 2021-05-27.03-02-42.mp4
> > Can't find a PW example but it's pretty much the same, I'd say revert to the old shutdown rate
> 
> Can this be looked at? The evidence is here.

not really, it seems to be related to increasing the maximum cruise ceiling above fl300
so given a choice between being able to cruise above fl300 or the engines taking slightly longer to stop spinning when they don't have fuel there's not much that can be done afaict.

If you can track down the exact change that caused it, and its not the bypass fan diameter then it is possible, but otherwise your guess is as good as mine.

**Comment by CalibraNR** [comment]
I was looking where to post it. Booth starts and shutdowns are way too slow. Booth N1 and N2 (GE) takes like ages to stop, more than 10min to drop below 5% N2.  Don`t know if anything can be done about it, since a lot of time has passed.

**Comment by CalibraNR** [comment]
@mSparks43 a nice video, show how fast EGT rises during start and the N1/N2 Spool Up time. https://www.youtube.com/watch?v=P6mvYyyMHV0

**Comment by CalibraNR** [comment]
I have literally ZERO knowledge on ACF files in XP. In any case, I used a similar aircraft with a GE CF6 engine and used it as a reference to adjust the Start-Spool time section of the 747-400F ACF file. Also, I ticked at the Engine Section engine controller prevents over-boost and engine controller prevents EGT over-temp. That solved the over tempo at start and accelerations, now (GE) if you fast forward the throttle it rises but stops at 751 Degrees C (EGT). Also, adjusted inside the Start/Spool section the engine friction to 2.000 and starter torque to 0.14 (think it can be adjusted a little more to speed up start). I will make a short flight to see how aircraft behaves in-flight, but it seems to improve A LOT the start and shutdown sequence. 

[747-400.zip](https://github.com/mSparks43/747-400/files/10786076/747-400.zip)
